apps/web/src/composables/auth.ts

```diff
@@ -33,7 +34,37 @@ export default function useAuth() {
         }
     }
 
+    function saveAccount(provider: ProviderString, address: string) {
```

Comment:
What do you think of `add/remove` or `save/delete`?

Can you handle a special case? If I add an account with a Ledger. Then tomorrow I add that same account through MetaMask (an account that is signing with that Ledger). Now I have the same account (address) in two places. We should only keep the latest account in this case (so remove the Ledger account and keep MetaMask-Ledger one). This could be a general rule (so beyond the Ledger and MetaMask case – anytime there is a duplicate account).

Reply:
@shanejearley - I resolved this in my latest commit. Did stumble upon a bug, but need to step away for a few and will revisit. Probably not even worth logging, but I'll just mention here that when connecting an account with MetaMask, the application is reading it as all lower case whereas connecting the same account with Ledger will contain some upper case characters in its address string...Probably an easy fix and will knock it out when I return to my laptop.

Reply:
That sounds right, Chris. I think ethers has relevant methods [here](https://docs.ethers.io/v5/api/utils/address/#utils-getAddress).

Reply:
@shanejearley - Yeah, so addresses are case-insensitive (at least for Ethereum), but there is a reason for the distinction having to do with checksums that I don't fully understand yet. Either way, for our save/remove purposes (in this PR), I just lowercase'd the address in order to standardize it.

PR should be good to merge (can remove the console.logs that show the save/remove functionality working; just let me know).